src/Collapse/CollapseGroup/CollapseGroup.component.tsx

```diff
@@ -0,0 +1,83 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled from 'styled-components';
+// COMPONENTS
+import { Themes } from '../Collapse.component';
+
+// The mundane function of CollapseGroup is to remove the bottom border of any Collapse children,
+// save for the last one.
+const StyledCollapseGroup = styled.div`
+    .anchor-collapse:not(:last-child) {
+        border-bottom-style: none;
+
+        & .anchor-collapse-button {
+            border-bottom-style: none;
+        }
+    }
+`;
+
+interface CollapseGroupProps {
+    /** Whether or not the CollapseGroup acts as an accordion, i.e. only one Collapse child can be open at a time. */
+    accordion?: boolean;
+    /** A zero-based index to indicate which Collapse child should be open first. Default is 0. */
+    openIndex?: number;
+    /** By passing a theme to the group, all Collapse children take on that theme */
+    theme?: Themes;
+    /** By passing an openIcon to the group, all Collapse children use that openIcon */
+    openIcon?: React.ReactElement;
+    /** By passing a closeIcon to the group, all Collapse children use that closeIcon */
+    closeIcon?: React.ReactElement;
+    // tslint:disable-next-line: max-line-length
+    /** Realistically anything can be a child of CollapseGroup, but it's rather pointless as it's intended to have Collapse components as children */
+    children?: any;
+    className?: string;
+}
+
+export const CollapseGroup = ({
+    accordion = false,
+    openIndex = 0,
+    theme,
+    openIcon,
+    closeIcon,
+    children,
+    className,
+}: CollapseGroupProps) => {
+    const [index, changeIndex] = React.useState<number>(
+        openIndex === 0 ? 0 : openIndex
+    );
+
+    return (
+        <StyledCollapseGroup
+            className={classNames('anchor-collapse-group', className)}
+        >
+            {
+                // tslint:disable-next-line: max-line-length
+                /** If accordion is true, all Collapse components are cloned in order to control their isOpen prop, and to use the changeIndex function to do so  */
+            }
+            {accordion
+                ? children &&
+                  children.map((child: any, i: number) => {
+                      return React.cloneElement(child, {
+                          isOpen: i === index,
+                          toggleFn: () => changeIndex(i),
```

Comment:
The way `accordion` is implemented currently, it isn't possible to close the currently open collapse so that everything is close. I was surprised by this when I was playing around on storybook.

Reply:
This is a **really** good point. I will work on adding that functionality.